.github/workflows/bors_ci.yml

```diff
@@ -69,225 +68,12 @@ jobs:
-          key: ${{ needs.cargo-registry-cache.outputs.cache-key }}
-
-      - name: Generate build artifacts key
-        id: cache-key
-        shell: bash
-        run: echo "::set-output name=key::$(echo "${{ runner.os }}-${{ steps.rust-toolchain.outputs.rustc }}-${{ github.job }}-${{ needs.cargo-registry-cache.outputs.cache-date }}-${{ hashFiles('**/Cargo.lock') }}")"
-
-      - name: Cache build artifacts
-        uses: actions/cache@c64c572235d810460d0d6876e9c705ad5002b353 # v2.1.6
-        with:
-          path: ${{ github.workspace }}/target
-          key: ${{ steps.cache-key.outputs.key }}
-          restore-keys: ${{ runner.os }}-${{ steps.rust-toolchain.outputs.rustc }}-${{ github.job }}-${{ needs.cargo-registry-cache.outputs.cache-date }}-
-
-      - name: Download data
-        run: sh download_data.sh
-
-      - name: Run tests
-        env:
-          RUSTFLAGS: "-D warnings"
-        run: |
-          cargo test --all-targets
-          cargo test --all-targets --all-features
-          cargo test --all-features --doc
-
-  cargo-tarpaulin:
-    name: cargo-tarpaulin
-    needs: [cargo-registry-cache, cargo-test]
-    runs-on: ubuntu-20.04
-    timeout-minutes: 30
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
-
-      - name: Install ${{ env.RUST_STABLE }} toolchain
-        id: rust-toolchain
-        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # v1.0.7
-        with:
-          toolchain: ${{ env.RUST_STABLE }}
-          profile: minimal
-          default: true
-
-      - name: Use cached cargo registry
-        uses: actions/cache@c64c572235d810460d0d6876e9c705ad5002b353 # v2.1.6
-        with:
-          path: |
-            ~/.cargo/registry/index
-            ~/.cargo/registry/cache
-            ~/.cargo/git/db
-          key: ${{ needs.cargo-registry-cache.outputs.cache-key }}
-
-      - name: Cache build artifacts
-        uses: actions/cache@c64c572235d810460d0d6876e9c705ad5002b353 # v2.1.6
-        with:
-          path: ${{ github.workspace }}/target
-          key: ${{ runner.os }}-${{ steps.rust-toolchain.outputs.rustc }}-${{ github.job }}-${{ needs.cargo-registry-cache.outputs.cache-date }}-${{ hashFiles('**/Cargo.lock') }}
-          restore-keys: ${{ runner.os }}-${{ steps.rust-toolchain.outputs.rustc }}-${{ github.job }}-${{ needs.cargo-registry-cache.outputs.cache-date }}-
-
-      - name: Download data
-        run: sh download_data.sh
-
-      - name: Run cargo-tarpaulin
-        uses: actions-rs/tarpaulin@044a1e5bdace8dd2f727b1af63c1d9a1d3572068 # v0.1.3
-        with:
-          version: '0.16.0'
-          args: '-v --all-features --force-clean --lib --ignore-tests --fail-under 70 --workspace --exclude xayn-ai-ffi-wasm --exclude dev-tool --timeout 300'
+  reusable-workflow:
+    uses: xaynetwork/xayn_ai/.github/workflows/ci_reusable_wf.yml@26482c656e6a079d3b1dd378ba4d0ca911b5a848
```

Comment:
Is it possible to use only the path (similar to calling composite actions) `./.github/workflows/ci_reusable_wf.yml`

Reply:
Unfortunately no, for now references to workflows must be prefixed with format 'owner/repository/'.

Reply:
is it also mandatory to put the commit there? We always want what is on the current branch

Reply:
Yes, the commit it is mandatory. Is either the commit hash or the version. This is the required syntax, it doesn't like changes, I tried.

{owner}/{repo}/{path}/{filename}@{ref}

Reply:
That means we always have to do two PRs to change something?

Reply:
Whenever you want to change something inside `ci_reusable_wf.yml`, unfortunately yes.